ext/js/language/translator.js

```diff
@@ -498,6 +490,23 @@ export class Translator {
         }
     }
 
+    /**
+     * @param {import('translation').FindTermsOptions} options
+     * @returns {Map<string, import('language').TextPreprocessor>}
+     */
+    _getTextPreprocessors(options) {
+        const {language} = options;
+        const textPreprocessorsData = getTextPreprocessors(language);
+
+        /** @type {Map<string, import('language').TextPreprocessor>} */
+        const textPreprocessors = textPreprocessorsData.reduce((map, preprocessor) => {
```

Comment:
Using .reduce to populate maps has always struct me as strange, maybe this is a personal thing, I dunno.

I'd suggest doing it like this, since it's slightly less code and also very marginally faster, although the speed probably doesn't matter in practice
```js
const textPreprocessors = new Map();
for (const preprocessor of textPreprocessorsData) {
    textPreprocessors.set(preprocessor.id, preprocessor);
}
```

chrome:
![image](https://github.com/themoeway/yomitan/assets/11037431/d229c938-c6cd-4a98-8148-dc96fce1ff49)

firefox:
![image](https://github.com/themoeway/yomitan/assets/11037431/52661f03-be3d-40b7-a093-c78b214760b3)


Oh also, this is how one of the maps is constructed above, so there's that also.